src/logics/compound-v3/tokens/data/mainnet.json

```diff
@@ -75,5 +75,12 @@
     "decimals": 18,
     "symbol": "wstETH",
     "name": "Wrapped liquid staked Ether 2.0"
+  },
+  "rETH": {
```

Comment:
Is this for lending-sdk?

Reply:
Removed. `rETH` is a collateral token built from `refresh-tokens-data.ts`, which we won't be using.